api/oapigen/types.go

```diff
@@ -130,7 +130,6 @@ type Task struct {
 	ModuleInput  *ModuleInput  `json:"module_input,omitempty"`
 	Name         string        `json:"name"`
 	Providers    *[]string     `json:"providers,omitempty"`
-	Services     *[]string     `json:"services,omitempty"`
```

Comment:
Since `task.services` is not supported in the API, could `task.condition` now be a required field?

Reply:
that's a really good point! I think it should be